zipline/_protocol.pyx

```diff
@@ -21,6 +21,28 @@ from cpython cimport bool
 
 from zipline.assets import Asset
 
+
+class assertKeywords(object):
```

Comment:
I don't think this falls into any of the python naming conventions, either `assert_keywords` or `AssertKeywords`.